src/iframemanager.js

```diff
@@ -857,6 +867,25 @@
 
         getConfig: () => config,
 
+        reset: () => {
+            clearObservers()
+
+            win = undefined
+            doc = undefined
+            config = undefined
+            services = undefined
+            onChangeCallback = undefined
+            allServiceProps = {}
+            serviceObservers = {}
+            stopServiceObserver = {}
+            currLang = ''
+            services = {}
+            serviceNames = undefined
```

Comment:
Using undefined causes some bugs. It should probably be best to set it to `[]`

These lines in the script `serviceNames.includes(serviceName)` will throw errors